.github/FUNDING.yml

```diff
@@ -1,2 +1 @@
-github: [johannestegner,frozzare] # rasmusbe <- FIXA FUNDING!
 open_collective: personnummer
```

Comment:
Not sure if this file is needed anymore since we have it here https://github.com/personnummer/.github

Reply:
That is true, Ill remove that one :)